common/lib/xmodule/xmodule/course_module.py

```diff
@@ -264,18 +267,32 @@ class CourseFields(object):
     )
     discussion_blackouts = List(
         display_name=_("Discussion Blackout Dates"),
-        help=_("Enter pairs of dates between which students cannot post to discussion forums. Each pair should be formatted as [\"YYYY-MM-DD\", \"YYYY-MM-DD\"]. To specify times as well as dates, format each pair as [\"YYYY-MM-DDTHH:MM\", \"YYYY-MM-DDTHH:MM\"] (be sure to include the \"T\" between the date and time). An entry defining more than one blackout period might look like this: [[\"2014-09-15\", \"2014-09-21\"], [\"2014-10-01\", \"2014-10-08\"]]"),
+        help=_(
+            "Enter pairs of dates between which students cannot post to discussion forums. Each pair should be "
+            "formatted as [\"YYYY-MM-DD\", \"YYYY-MM-DD\"]. To specify times as well as dates, format each pair "
+            "as [\"YYYY-MM-DDTHH:MM\", \"YYYY-MM-DDTHH:MM\"] (be sure to include the \"T\" between the date and "
+            "time). An entry defining more than one blackout period might look like this: "
+            "[[\"2014-09-15\", \"2014-09-21\"], [\"2014-10-01\", \"2014-10-08\"]]"
```

Comment:
@caesar2164 I don't think that's the right solution, because the double quotes may have semantic meaning.

Actually... will it work with i18n if we wrap the translation string in single quotes and then don't escape the double quotes? I think it would.... can you try pushing that up, for this and the next string (and any other strings like that in this file) and I'll test the i18n and see if it scrapes and delivers properly on your branch?